drivers/sensor/tdk/icp101xx/EmbUtils/InvExport.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#ifndef _INV_IDD_EXPORT_H_
+#define _INV_IDD_EXPORT_H_
+
+#if defined(_WIN32)
```

Comment:
When would we be building this for windows as a shared lib

Reply:
The goal of his PR is to make sure Zephyr driver for ICP101xx sensors is using the official driver from TDK.
This code is part of this official TDK driver and cannot be modified for Zephyr.